resources/dicts/events/death/beach.json

```diff
@@ -1,35 +1,32 @@
 [
     {
-        "event_id": "bch_death_",
+        "event_id": "bch_death_drown_any1",
+        "biome": ["beach"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
         "tags": [
-            "no_body"
+            "no_body",
+            "all_lives"
         ],
         "weight": 20,
         "event_text": "m_c was washed out to sea, never to be seen again.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": ["any"],
```

Comment:
might consider restricting this so that newborns and kittens don't receive this event